packages/react-cosmos-redux-proxy/src/index.jsx

```diff
@@ -19,35 +28,68 @@ const ReduxProxyFactory = ({
       };
     }
 
+    componentWillMount() {
+      const {
+        store,
+        onStoreChange,
+      } = this;
+      if (store) {
+        this.storeUnsubscribe = store.subscribe(onStoreChange);
+      }
+    }
+
+    componentWillUnmount() {
+      if (this.storeUnsubscribe) {
```

Comment:
Why? It's a private affair, no body is gonna set an `storeUnsubscribe` property of a different type...